### DISCUSSION TITLE
Replace axios with http in auth module

### DISCUSSION THREAD

**Discussion by max5432112345** [Teranode/nuxt-module-alternatives#65] [General]
Why if I try to replace axios with http in auth module the line of code
`    console.log(await this.ctx.$http.$get(_endpoint.url));
`
return me not response but 
```
baseURL: "http://localhost:3000"
credentials: "omit"
headers: {}
method: "get"
retry: false
timeout: 25000
url: "/api/auth/login"
[[Prototype]]: Object
```

**Comment by Denoder** [comment]
I have transitioned the module to use ohmyfetch now.
